tests/system/action/chat_group/test_create.py

```diff
@@ -5,7 +5,10 @@
 class ChatGroupCreate(BaseActionTestCase):
```

Comment:
Add a test where `meeting/enable_chat` is true, but `organization/enable_chat` is false and assert it fails

Reply:
Done.